t/cmd/lfstest-gitserver.go

```diff
@@ -749,6 +749,24 @@ func storageHandler(w http.ResponseWriter, r *http.Request) {
 					byteLimit = 8
 					batchResumeFailFallbackStorageAttempts++
 				}
+			} else if len(by) == len("status-batch-retry") && string(by) == "status-batch-retry" {
```

Comment:
We may be copying what's already here, but is there a need to check the string both for string and length equality? Shouldn't the later follow from the former?

Reply:
I think this was an optimization (perform the cheap check first), but I'm happy to just remove it and switch to a string comparison. I expect Go does the same comparison internally anyway.